tests/foreman/virtwho/cli/test_esx_sca.py

```diff
@@ -669,3 +669,36 @@ def test_positive_post_hypervisors_with_fake_different_org_simultaneous(
         for item in task:
             assert "Job blocked by the following existing jobs" not in item['task-errors']
             assert "success" in item['result']
+
+
+@pytest.mark.parametrize('deploy_type_cli', ['id'], indirect=True)
+def test_positive_change_system_puropse_SLA_for_hypervisor(
+    target_sat, virtwho_config_cli, deploy_type_cli
+):
+    """Verify that system purpose SLA attribute set successfully and does not throw any error
+
+    :id: a4c88d9f-35b7-4073-a43c-ab613f4ce583
+
+    :setup:
+        Setup vir-who hypervisor
+
+    :steps:
+        Set up system purpose attribute for SLA
+
+    :expectedresults:
+        system purpose SLA attribute set without any error
+
+    :BZ: SAT-28552
+
+    :customerscenario: true
+
+    :CaseImportance: Normal
```

Comment:
test_positive_change_system_puropse_SLA_for_hypervisor:675
----------------------------------------------------------

* Tokens with invalid values:
  Caseimportance: Normal
      type: choice
      case sensitive: True
      choices: ['Critical', 'High', 'Medium', 'Low']

Choose proper CaseImportance so that code check is happy please

Reply:
I don't think we do :D

Reply:
We used to take them from Bug severity. SAT-28552 is _Normal_ which I would map as _Medium_ probably.